build/builder/Dockerfile

```
@@ -251,7 +251,8 @@ RUN apt-get install -y --no-install-recommends \
     openssh-client \
     yarn \
     google-chrome-stable \
-    unzip
+    unzip \
+    && cp /usr/bin/bazel-3.7.0 /usr/bin/bazel
```

How about we link it instead? Save some disk space. (It's what I'm building with.)

```suggestion
    && ln -s /usr/bin/bazel-3.7.0 /usr/bin/bazel
```